interceptors/logging/options.go

```diff
@@ -204,3 +206,21 @@ func WithTimestampFormat(format string) Option {
 		o.timestampFormat = format
 	}
 }
+
+// AddGrpcLogFields customizes the function for adding gRPC fields to the log entry from the below list.
+//
+//	-SystemTag[0],
+//	-ComponentFieldKey,
+//	-ServiceFieldKey,
+//	-MethodFieldKey,
+//	-MethodTypeFieldKey,
+//
+// If not provided default fields will be used.
+type AddGrpcLogFields func() []string
+
+// WithAddGrpcLogFields customizes the function for adding gRPC fields to the log entry.
+func WithGrpcLogFields(f AddGrpcLogFields) Option {
+	return func(o *options) {
+		o.grpcLogFields = f
+	}
+}
```

Comment:
This seems like a complicated way to ignore fields from the default fields. I think just a way to pass the existing field names might be easier?

```go
func WithDisableLoggingField(disabledLogFields ...string) Option {
	return func(o *options) {
		o.disabledLogFields = disabledLogFields
	}
```

Then if can be used like so:

```go
[]grpc.DialOption{
		grpc.WithUnaryInterceptor(logging.UnaryClientInterceptor(s.logger, logging.WithGrpcLogFields(logging.MethodFieldKey, logging.MethodTypeFieldKey))),
	}
```

What do you think?

Reply:
Yes, that would be better.  Ive updated the commit. :)